### PR TITLE
Mark

### PR DESCRIPTION
**Pull request by MarkLTurner**

### CODE REVIEW

**Review by alisonrclarke — CHANGES_REQUESTED** [review]
I don't think #31 should be closed until we've actually run Black over the python code, so please could you edit the description so it doesn't say 'fixes'?

**Review by alisonrclarke — COMMENTED** [review]
Please could you remove the .pyc files? Probably worth adding the contents of [python.gitignore](https://github.com/github/gitignore/blob/master/Python.gitignore) to our .gitignore file as well to avoid committing them by accident in future.

**Comment by alisonrclarke** [comment]
> Please could you remove the .pyc files? Probably worth adding the contents of [python.gitignore](https://github.com/github/gitignore/blob/master/Python.gitignore) to our .gitignore file as well to avoid committing them by accident in future.

Sorry, I see you've already done this - that'll teach me to click on the link in the most recent email!

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - defaults
+dependencies:
+  - black
+  - conda-lock
+  - pre_commit
+  - mesa
+  - numpy
+  - pandas
+  - scipy
+prefix: /opt/homebrew/Caskroom/miniconda/base/envs/mesa_env
```
Comment (alisonrclarke) [inline environment.yml:14]: I think we can remove the `prefix` line as it maps to a path that's specific to your machine.

**Review by alisonrclarke — APPROVED** [review]
This is looking good now - thanks for making the changes! There's one comment about the environment.yml you might want to address before merging but it's not a deal-breaker - feel free to merge!

**Comment by MarkLTurner** [comment]
> This is looking good now - thanks for making the changes! There's one comment about the environment.yml you might want to address before merging but it's not a deal-breaker - feel free to merge!

Great! 'Prefix' has been removed. Thanks for reviewing.

**Pull request merged**